### PR TITLE
fix the no fire sound problem

### PR DESCRIPTION
**Pull request by Alexander5421**
try to solve #125

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize?pullRequest=126) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize?pullRequest=126) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.multiplayer.samples.bitesize?pullRequest=126) it.</sub>

File: Basic/2DSpaceShooter/Assets/Scripts/ShipControl.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Runtime.CompilerServices;
```
Comment (fernando-cortez) [inline Basic/2DSpaceShooter/Assets/Scripts/ShipControl.cs:2]: This directive can be removed!

File: Basic/2DSpaceShooter/Assets/Scripts/ShipControl.cs [hunk 2]
```diff
@@ -537,7 +545,7 @@ public void FireServerRpc()
             }
         }
     }
-
+
```
Comment (fernando-cortez) [inline Basic/2DSpaceShooter/Assets/Scripts/ShipControl.cs:548]: This extra space should be removed!

**Review by fernando-cortez — CHANGES_REQUESTED** [review]
Looks good besides the changes I've mentioned!

**Comment by fernando-cortez** [comment]
Hi @Alexander5421, this PR looks good. However, if your PR could target our develop branch, that would be greatly appreciated.

Having the fire sound be played on the invocation of a ClientRpc is a good fix!

**Comment by Alexander5421** [comment]
Hi @fernando-cortez, I created a new pull request #138  and merged it into develop instead of the main

**Comment by fernando-cortez** [comment]
@Alexander5421 
Thanks again for your contribution.
I'm closing this PR as your latest PR has been integrated into one of our staging branches [here](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/pull/148).

**Pull request closed without merging**